src/units/units.cpp

```diff
@@ -314,10 +310,9 @@ void UnitTable::print_units_sorted(std::ostream& units_details) const {
                                                                                table.end());
     std::sort(sorted_elements.begin(), sorted_elements.end());
     for (const auto& it: sorted_elements) {
-        units_details << fmt::format("{} {:.{}f}: {}\n",
```

Comment:
Instead of being based to a default print scheme of fmt I think it would be better to use `{:g}` instead to be consistent with other places we print float numbers as well